client/dashboard/sites/logs/index.tsx

```diff
@@ -71,6 +93,11 @@ function SiteLogs( { logType }: { logType: LogType } ) {
 				asOverlay
 				{ ...getLogsCalloutProps() }
 			>
+				{ autoRefreshDisabledReason && (
+					<div style={ { marginBottom: 12 } }>
```

Comment:
Please use `<VStack>` rather than custom margin. We can simply replace the fragment on line 101 with this.